cmd/license-header/main.go

```diff
@@ -177,7 +177,7 @@ func run(ctx context.Context, container app.Container, flags *flags) error {
 						return err
 					}
 					if flags.ExitCode {
-						return app.NewError(100, "")
+						return app.NewError(100, "") //nolint:mnd
```

Comment:
We already have an ignore for "gomnd" in the config file, which is basically this same check (and, from the name, seems redundant 🤷). So I'd add "mnd" to the list of ignored checkers above instead of using nolint comments.